jsapp/js/constants.es6

```diff
@@ -70,6 +70,7 @@ export const MODAL_TYPES = {
   ASSET_TAGS: 'asset-tags',
   ENCRYPT_FORM: 'encrypt-form',
   BULK_EDIT_SUBMISSIONS: 'bulk-edit-submissions',
+  DATA_ATTACHMENT_COLUMNS: 'data-attachment-columns',
```

Comment:
Should there also be an `Object.freeze()` here?

Reply:
cc @magicznyleszek :D

Reply:
Yeah let's add it :) Now I freeze every `const` object I define :)